taurus/demo/cake.py

```diff
@@ -306,11 +401,14 @@ def make_cake_spec():
             tags=[
                 'purchase::dry-goods'
             ],
-            notes='Purchasing salt'
+            notes='Purchasing salt',
```

Comment:
Why is this comma here?

Reply:
Good catch, I think, thought it's not clear why flake didn't catch this
```suggestion
            notes='Purchasing salt'
```